keyring/system.go

```diff
@@ -53,3 +53,9 @@ func (s *systemKeyring) Get(name string) ([]byte, error) {
 func (s *systemKeyring) Delete(user string) error {
 	return keyring.Delete(s.service, user)
 }
+
+func (s *systemKeyring) List() ([]string, error) {
+	// List does not need to be returned here
+	// This function is a stub
```

Comment:
suggestion: It would be nice to have a bit more details as to why list does not need to be returned for the system keyring.

Reply:
This seems like a fair suggestion. I agree with it. I've added a bit more to it.

I'm not sure how much detail you (or other team members(?)) think this comment should go into. I'm open to discussion on this point if anyone feels strongly.